lib/serverspec/type/hdfs_config.rb

```diff
@@ -0,0 +1,22 @@
+begin
+  require 'nokogiri'
+rescue LoadError
+  fail "nokogiri is not available. Try installing it."
+end
```

Comment:
How about this like #548 ?

```ruby
describe hdfs_config('dfs.nameservices', :file => '/path/to/file') do
    its(:value) { should eq "test1" }
  end
```

You should rebase your branch on latest master to implement this.